instructions-for-devs.md

```diff
@@ -0,0 +1,174 @@
+# Instructions for Developers
+
+This document contains `xud` setup instructions for developers and everyone with a need for more fine-grained control when running `xud`.
+
+## 1. Install
+
+First, clone the repository from GitHub and install dependencies. 
+
+Developers:
+```bash
+git clone https://github.com/ExchangeUnion/xud
+cd xud
+npm install
+```
+For everyone else, we recommend cloning the [latest release](https://github.com/ExchangeUnion/xud/releases):
+```bash
+git clone https://github.com/ExchangeUnion/xud -b 1.0.0-prealpha
+cd xud
+npm install
+```
+
+
+Xud uses [MySQL](https://www.mysql.com/) or [MariaDB](https://mariadb.org/). You will have to install one of those and create a user *xud* and a database *xud* and grant all permissions for the new database to the *xud* user.
+
+## 2. Prepare your environment 
+
+To initialize the database with some testing data, run the following command from `~/xud`:
+
+```bash
+npm run db:init
+```
+
+Setup raiden according to [this guide](https://github.com/ExchangeUnion/xud/blob/raiden_swap/lib/raidenclient/README.md).
```

Comment:
We have to update this link when the Raiden PR gets merged.

Reply:
We will.